src/modules/weather/Weather.ts

```diff
@@ -32,37 +32,32 @@ export default class Weather {
-                [{ type: PokemonType.Electric, multiplier: 1.1 }, { type: PokemonType.Water, multiplier: 1.1 }, { type: PokemonType.Fire, multiplier: 0.9 }]),
+            new WeatherCondition(WeatherType.Thunderstorm, '#a19288', 'It\'s currently raining heavily with thunder.', 10,
+                [{ type: PokemonType.Electric, multiplier: 1.1 }, { type: PokemonType.Water, multiplier: 1.10 }, { type: PokemonType.Dragon, multiplier: 1.1 }]),
         [WeatherType.Snow]:
-            new WeatherCondition(WeatherType.Snow, '#bbe6e6', 'It\'s cold and snowing.', 5,
-                [{ type: PokemonType.Ice, multiplier: 1.05 }]),
-        [WeatherType.Hail]:
-            new WeatherCondition(WeatherType.Hail, '#74e6e6', 'It\'s cold and hailing.', 3,
+            new WeatherCondition(WeatherType.Snow, '#bbe6e6', 'It\'s cold and snowing.', 10,
                 [{ type: PokemonType.Ice, multiplier: 1.1 }]),
         [WeatherType.Blizzard]:
-            new WeatherCondition(WeatherType.Blizzard, '#98d8d8', 'A howling blizzard blows.', 2,
-                [{ type: PokemonType.Ice, multiplier: 1.2 }, { type: PokemonType.Fire, multiplier: 0.9 }, { type: PokemonType.Grass, multiplier: 0.9 }]),
+            new WeatherCondition(WeatherType.Blizzard, '#98d8d8', 'A howling blizzard blows.', 10,
+                [{ type: PokemonType.Ice, multiplier: 1.1 }, { type: PokemonType.Steel, multiplier: 1.1 }]),
         [WeatherType.Sunny]:
             new WeatherCondition(WeatherType.Sunny, '#f5ac78', 'The sunlight is strong.', 10,
-                [{ type: PokemonType.Fire, multiplier: 1.2 }, { type: PokemonType.Grass, multiplier: 1.1 }, { type: PokemonType.Water, multiplier: 0.9 }]),
+                [{ type: PokemonType.Fire, multiplier: 1.1 }, { type: PokemonType.Ground, multiplier: 1.1 }]),
         [WeatherType.Sandstorm]:
             new WeatherCondition(WeatherType.Sandstorm, '#d1c07d', 'A sandstorm is raging.', 10,
-                [{ type: PokemonType.Rock, multiplier: 1.1 }, { type: PokemonType.Ground, multiplier: 1.1 }, { type: PokemonType.Steel, multiplier: 1.1 }]),
+                [{ type: PokemonType.Rock, multiplier: 1.1 }, { type: PokemonType.Ground, multiplier: 1.1 }]),
         [WeatherType.Fog]:
             new WeatherCondition(WeatherType.Fog, '#d2c2ef', 'The fog is deep...', 10,
-                [{ type: PokemonType.Ghost, multiplier: 1.2 }, { type: PokemonType.Dark, multiplier: 1.1 }, { type: PokemonType.Electric, multiplier: 0.9 }]),
-        [WeatherType.Windy]:
-            new WeatherCondition(WeatherType.Windy, '#81c4ca', 'Mysterious strong winds blow.', 1,
-                [{ type: PokemonType.Flying, multiplier: 1.2 }, { type: PokemonType.Dragon, multiplier: 1.1 }]),
+                [{ type: PokemonType.Psychic, multiplier: 1.1 }, { type: PokemonType.Ghost, multiplier: 1.1 }, { type: PokemonType.Fairy, multiplier: 1.1 }]),
```

Comment:
```suggestion
                [{ type: PokemonType.Psychic, multiplier: 1.1 }, { type: PokemonType.Ghost, multiplier: 1.1 }, { type: PokemonType.Fairy, multiplier: 1.1 }, { type: PokemonType.Electric, multiplier: 0.9 }]),
```

Reply:
Restores Electric debuff

Reply:
If windy is being kept then a re-review of weather buffs would be needed as the original changes removes windy